tmpy/session/tm_sage.py

```diff
@@ -24,38 +24,34 @@
 import traceback
 import re
 import string
+import ast
 import warnings
 warnings.simplefilter("ignore") # don't print warnings to stdout
 from tmpy.compat import py_ver
-from tmpy.protocol   import *
-from tmpy.postscript import ps_out
+from tmpy.capture import CaptureStdout
+from tmpy.postscript import ps_out, PSOutDummy, pdf_out, FileOutDummy
 from tmpy.completion import parse_complete_command, complete
+from tmpy.protocol   import *
 from sage.all        import *
 
-__version__ = '0.8.1'
-__author__ = 'Ero Carrera'
+if py_ver == 2:
+    flush_err ("Python 2 is no longer supported, please use Python 3")
+    exit (-1)
```

Comment:
It hurts the old SageMath user. Imagine a SageMath and TeXmacs user who use an old Ubuntu release shipped with old SageMath which does not support Python 3.